### ISSUE TITLE
Create a version of the PDF that uses OpenDyslexic

### ISSUE THREAD

**Issue by ralienpp** [ralienpp/book-one#9]
https://github.com/antijingoist/opendyslexic

Add a boolean switch that takes care of it, and saves the result to another PDF, e.g. main-dyslexic.pdf

Minimal example

```
\documentclass[12pt]{article}

\title{Using Dyslexic-Friendly Fonts on Overleaf}

\usepackage{fontspec}
\setmainfont{OpenDyslexic}
% \setmainfont{OpenDyslexicAlta}  %% uses a different "a"

\usepackage{lipsum}
\begin{document}
\lipsum[1]
\end{document}
```

**Comment by ralienpp** [comment]
This works out of the box with XeLaTeX, but that's not what I've got here.

Figure out how to convert OTF to something pdflatex can use, see https://www.overleaf.com/learn/latex/Questions/I_have_a_custom_font_I%27d_like_to_load_to_my_document._How_can_I_do_this%3F#Using_custom_fonts_with_pdfLaTeX

**Comment by ralienpp** [comment]
Even though I could compile it with Luatex, OpenDyslexic doesn't have all the Cyrillic glyphs https://github.com/antijingoist/opendyslexic/issues/6

This remains an open issue for now; consider using LexiaD or some other alternative.

**Comment by ralienpp** [comment]
LexiaD is not available for download, try this one instead: https://adysfont.com/

**Comment by ralienpp** [comment]
It worked out with Adys; other fonts don't have the required glyphs. Note: this version can only be built with `lualatex`.

Resolved in #12
